mypyc/external/googletest/include/gtest/internal/gtest-internal.h

```diff
@@ -1235,4 +1235,3 @@ ::testing::TestInfo* const GTEST_TEST_CLASS_NAME_(test_case_name, test_name)\
 void GTEST_TEST_CLASS_NAME_(test_case_name, test_name)::TestBody()
 
 #endif  // GTEST_INCLUDE_GTEST_INTERNAL_GTEST_INTERNAL_H_
-
```

Comment:
This is third-party code, so modifying it may make it slightly harder to update it in the future.

Reply:
Yeah, I wondered about that.

Reply:
Addressed in 60cb8f15815ce5f77742a61bf4f13eec53400152